There has been a change in the method snapshot, in the section of code starting on line nr 207.
  
The change is in a catch statement, in a try statement, in a synchronized method, in the method ```snapshot```, in the class ```FsTranslog```.
  
The following changes have been made:  
Changed type reference from ```java.io.IOException``` to ```Exception``` on line 207.  
